frontend/src/services/routes/routes.ts

```diff
@@ -4,6 +4,7 @@ export const routes = {
 		register: '/account/register',
 		login: '/account/login',
 		profile: '/account/profile',
+		forgotPassword: '/account/forgot-password',
```

Comment:
ich habe einen interessanten bug... wenn ich auf diese url klicke im gleiche window bleibe wird meine url zu: `/account/forgot-pass` sehr interessant... außerdem würde ich das nicht so nennen

Reply:
done in https://github.com/dopeshot/beyond-life/pull/169/commits/cd13a1cbfef561b5c6247d28e434bc7b176bc66c